catalog/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "tagline": "Identify right-sizing opportunities and potential savings of your AWS, GCP, and Azure instances across your cloud environment",
+  "repository": "https://github.com/newrelic/nr1-cloud-optimize.git",
+  "details": "Cloud Optimize analyzes your cloud environment using the New Relic Infrastructure cloud integrations. \n The application compares the size of your instances to their utilization, identifying resources that are sized larger than needed. Cloud Optimize will estimate your savings by optimizing resource size.\n Users of Cloud Optimize are able to select the hosts, regions and other configurations to specify their unique business use cases.\n This application supports AWS, GCP, and Azure cloud infrastructures.",
```

Comment:
I think putting two `\n\n` will create the space between paragraphs that we're looking for, but I might have that wrong (I used `\n\n` in the PRs I did, but maybe I need to review as well). We should probably confirm with devex team to be sure.